File: Python/UPCMatricula/models/Curso.py
class Curso(object):
    def __init__(self, nombre):
        self.nombre = nombre
        self.alumnos = []

    def agregar_alumnos(self, alumno):
        self.alumnos.append(alumno)

    def listar_matriculados(self):
        listado = "Curso: " + self.nombre + "\n\n"
        for alumno in self.alumnos:
            listado += str(alumno) + "\n"
        return listado

File: Ruby/UPCMatricula/models/Curso.rb
class Curso

    def initialize(nombre)
        @nombre = nombre
        @alumnos = []
    end

    def agregar_alumno(alumno)
        @alumnos += [alumno]
    end

    def listar_matriculados
        listado = "Curso: " + @nombre + "\n\n"
        @alumnos.each do |alumno|
            listado += alumno.to_s + "\n"
        end
        return listado
    end

end
